config/zones.json

```
@@ -5073,16 +5074,18 @@
       ]
     ],
     "capacity": {
-      "biomass": 257,
-      "gas": 10344,
-      "solar": 231.26
+      "biomass": 256.8,
+      "gas": 10671.41,
+      "solar": 290.167862138,
+      "hydro": 1363.6
```

These hydro generators must have some Tardis qualities to fit 1 GW of hydro in among 5+ million people in 729 km^2 :laughing: 

Looks like this 1363.6 MW figure is "steam turbine" in the source. In the parser, we assign steam turbines to "unknown", because per the data we had in 2016, steam turbines in Singapore "typically run on fuel oil and diesel" (see https://github.com/tmrowco/electricitymap-contrib/blob/master/parsers/SG.py). So let's add it as the "unknown".

```suggestion
      "unknown": 1363.6
```